src/bin/find.js

```diff
@@ -8,6 +8,7 @@ var options = {
   getAllAvailableRules: ['all-available', 'a'],
   getUnusedRules: ['unused', 'u'],
   n: ['no-error'],
+  noCore: ['nocore', 'C'],
```

Comment:
may be change 
`n: ['no-error']` to `ne: ['no-error'],` or `noError: ['no-error']`,
`noCore: ['nocore']` to `nc: ['no-core'],` or `noCore: ['no-core'],`